tests/books/test_edit_books.py

```diff
@@ -0,0 +1,60 @@
+import assertpy
+import unittest
+
+from assertpy import assert_that
+
+from actions.book_endpoint_actions import (do_delete_request_for_book,
+                                           do_post_request_to_create_book,
+                                           do_put_request_to_update_book)
+from helpers.books_helper import append_to_books_list
+from models.books_model import get_valid_minim_required_create_book_payload
+from tests import logger, fake
+
+
+class EditBookTest(unittest.TestCase):
+
+    def setUp(self):
+        self.book_ids = []
+
+    def tearDown(self):
+        self.all_book_ids_deleted = True
+        for book_id in self.book_ids:
+            response = do_delete_request_for_book(book_id)
+            if not response.ok:
+                self.all_book_ids_deleted = False
+                logger.debug('Not all books were deleted')
+        assertpy.assert_that(self.all_book_ids_deleted).described_as("Not all books were deleted").is_true()
+
+    def test_edit_book_details(self):
+        request_body = get_valid_minim_required_create_book_payload()
+        response_post = do_post_request_to_create_book(request_body)
+        self.book_ids = append_to_books_list(response=response_post, book_ids=self.book_ids)
+        new_title = fake.text(20)
+        new_author = fake.name()
+        request_body['name'] = new_title
```

Comment:
you could create a method for update_book_payload(**kwars), iterate them and change them. then use the method like:  update_book_payload(name="asdad", author="dadsaa")